lib/trento_web/openapi/v1/schema/user.ex

```diff
@@ -165,6 +170,11 @@ defmodule TrentoWeb.OpenApi.V1.Schema.User do
             type: :boolean,
             description: "Whether user analytics collection is enabled",
             nullable: false
+          },
+          analytics_eula_accepted: %Schema{
+            type: :boolean,
+            description: "Whether user analytics eula is accepted",
```

Comment:
nitpick:
maybe here as well, in the `description`, we could say that setting this value to `false` has no effect